GSA_Adapter/CRUD/Read.cs

```diff
@@ -310,16 +313,16 @@ private List<IMaterialFragment> GetStandardGsaMaterials()
             List<string> names = new List<string> { "STEEL", "CONC_SHORT", "CONC_LONG", "ALUMINIUM", "GLASS" };
 
             List<IMaterialFragment> materials = new List<IMaterialFragment>();
-            materials.Add(new Steel() { Name = "GSA Standard STEEL", CustomData = new Dictionary<string, object> { { AdapterIdName, "STEEL" } } });
-            materials.Add(new Concrete() { Name = "GSA Standard CONC_SHORT", CustomData = new Dictionary<string, object> { { AdapterIdName, "CONC_SHORT" } } });
-            materials.Add(new Concrete() { Name = "GSA Standard CONC_LONG", CustomData = new Dictionary<string, object> { { AdapterIdName, "CONC_LONG" } } });
-            materials.Add(new Aluminium() { Name = "GSA Standard ALUMINIUM", CustomData = new Dictionary<string, object> { { AdapterIdName, "ALUMINIUM" } } });
+            materials.Add(new Steel() { Name = "STEEL" });
+            materials.Add(new Concrete() { Name = "CONC_SHORT" });
+            materials.Add(new Concrete() { Name = "CONC_LONG" });
+            materials.Add(new Aluminium() { Name = "ALUMINIUM" });
```

Comment:
Goes together with the change for reading sections and surface properties
```suggestion
            materials.Add(new Steel() { Name = "STEEL" });
            materials.Add(new Concrete() { Name = "CONC_SHORT" });
            materials.Add(new Concrete() { Name = "CONC_LONG" });
            materials.Add(new Aluminium() { Name = "ALUMINIUM" });

            foreach (IMaterialFragment material in materials)
            {
                SetAdapterId(material, material.Name);
            }
```